src/avram/criteria.cr

```diff
@@ -154,6 +154,11 @@ class Avram::Criteria(T, V)
     add_clause(Avram::Where::In.new(column, values))
   end
 
+  def have_any?(values) : T
```

Comment:
I like the name better than my original idea; however, my only (very very small) concern is would this be confusing that it's a predicate method that doesn't return Bool or denote a nilable value? Maybe it's fine in this case, but I don't want people to confuse it with the `none?` or `any?` [methods](https://github.com/luckyframework/avram/blob/1326a8ea7583b91169993194802bb0875172a9c3/src/avram/queryable.cr#L39-L45) we have.

```crystal
if UserQuery.new.tags.have_any?(["featured"])
  # would always return truethy
end

# would this be strange? lol
UserQuery.new.tags.have_any?(["featured"]).any?
```

But if you don't think it'll be that big of an issue with some proper docs, then I'm cool with it as is :+1:

Reply:
What about just `have_any` without the question mark?